addons/hr_holidays/tests/test_accrual_allocations.py

```diff
@@ -45,207 +45,3 @@ def set_employee_create_date(self, id, newdate):
-            'interval_number': 1,
-            'unit_per_interval': 'days',
-            'interval_unit': 'weeks',
-        })
-
-        alloc_0.action_approve()
-
-        Allocation._update_accrual()
-
-        self.assertEqual(alloc_0.number_of_days, 0, 'Employee is new he should not get any accrual leaves')
-
-    def test_accrual_multi(self):
-        """ Test if the cron does not allocate leaves every time it's called but only when necessary """
-        alloc = self.env['hr.leave.allocation'].with_user(self.user_hrmanager_id).with_context(tracking_disable=True).create({
-            'name': '2 days per week',
-            'employee_id': self.employee_emp_id,
-            'holiday_status_id': self.accrual_type.id,
-            'allocation_type': 'accrual',
-            'number_of_days': 0,
-            'number_per_interval': 1,
-            'interval_number': 2,
-            'unit_per_interval': 'days',
-            'interval_unit': 'weeks',
-        })
-        alloc.action_approve()
-        alloc._update_accrual()
-        alloc._update_accrual()
-
-        self.assertEqual(alloc.number_of_days, 1, 'Cron only allocates 1 days every two weeks')
-
-    def test_accrual_validation(self):
-        """
-            Test if cron does not allocate past it's validity date
-        """
-        Allocation = self.env['hr.leave.allocation'].with_user(self.user_hrmanager_id).with_context(tracking_disable=True)
-
-        alloc_0 = Allocation.create({
-            'name': '20 days per year',
-            'employee_id': self.employee_emp_id,
-            'holiday_status_id': self.accrual_type.id,
-            'number_of_days': 0,
-            'number_per_interval': 20,
-            'interval_number': 1,
-            'unit_per_interval': 'days',
-            'interval_unit': 'years',
-            'date_to': fields.Datetime.from_string('2015-02-03 00:00:00'),
-        })
-
-        alloc_0.action_approve()
-
-        Allocation._update_accrual()
-
-        self.assertEqual(alloc_0.number_of_days, 0, 'Cron validity passed, should not allocate any leave')
-
-    def test_accrual_balance_limit(self):
-        """ Test if accrual allocation does not allocate more than the balance limit"""
-        allocation = self.env['hr.leave.allocation'].with_user(self.user_hrmanager_id).with_context(tracking_disable=True).create({
-            'name': 'accrual 5 max',
-            'employee_id': self.employee_emp_id,
-            'holiday_status_id': self.accrual_type.id,
-            'allocation_type': 'accrual',
-            'accrual_limit': 5,
-            'number_of_days': 0,
-            'number_per_interval': 6,
-            'interval_number': 1,
-            'unit_per_interval': 'days',
-            'interval_unit': 'weeks',
-        })
-        allocation.action_approve()
-        allocation._update_accrual()
-
-        self.assertEqual(allocation.number_of_days, 5, 'Should have allocated only 5 days as balance limit is 5')
```

Comment:
why did you completely remove these tests instead of adapting them to the new system ?